src/Zadanie16.java

```diff
@@ -0,0 +1,9 @@
+public class Zadanie16 {
+    public static void main(String[] args) {
+        int sum = 0;
+        for(int x=-30;x<=1566;sum += x++) {
```

Comment:
podoba mi się ten zapis ale dla kogoś innego może być mało czytelny, zrobiłbym w for takie coś:
for()
{
sum += x;
}